content/docs/apps/custom-domains.md

```diff
@@ -10,7 +10,10 @@ Here's how to put your app on your project's custom domain name. For context, [C
 The [Manual Method](#manual-method) is available to all users.
 
 {{% govcloud %}}
-We've also developed a [Managed Service Method](#managed-service-method).
+We've also developed a [Managed Service Method](#managed-service-method). This option is vastly preferred, as it
+
+* Is (somewhat) self-service
```

Comment:
The broken sentence and initial upcasing is confusing. Let's make it one sentence with no bullets.